README.md

```diff
@@ -108,6 +108,26 @@ adb push tests/tests /data/local/tmp
 adb shell /data/local/tmp/tests
 ```
 
+## Using Docker
+
+The `Dockerfile` included in this repo will provision a Docker image suitable
+for building and running tests for both the Linux and Android platforms.
+
+```
+# Build Docker image from Dockerfile
+docker build -t "objectstore" .
+# Run bash interactively from the built Docker image,
+# mounting the current directory
+docker run -it -v `pwd`:`pwd` -w `pwd` objectstore bash
+# Build the object store for Linux and run tests
+> cmake .
```

Comment:
Elsewhere in the object store README we assume that you're building in the root of the source directory. Using a different directory also works, but makes the instructions more convoluted than they need to be.